src/screens/Settings/components/GeneralSettings/index.tsx

```diff
@@ -252,6 +256,12 @@ export default function GeneralSettings({
           <ToggleSwitch value={exitToTray} handleChange={toggleTray} />
         </span>
       </span>
+      { exitToTray && <span className="setting">
```

Comment:
A new test would be great for that. Testing that this is only rendered if exitToTray is true.

Reply:
Good idea, I added a couple of tests for showing and hiding the "start minimized" option